.github/workflows/build-linux-installer-deb.yml

```diff
@@ -154,14 +154,36 @@ jobs:
       run: |
         sudo apt-get install -y jq
 
+    - name: Prepare GUI cache
+      id: gui-ref
+      run: |
+        gui_ref=$(git submodule status chia-blockchain-gui | sed -e 's/^ //g' -e 's/ chia-blockchain-gui.*$//g')
+        echo "${gui_ref}"
+        echo "::set-output name=GUI_REF::${gui_ref}"
+        echo "rm -rf ./chia-blockchain-gui"
+        rm -rf ./chia-blockchain-gui
+
+    - name: Cache GUI
+      uses: actions/cache@v3
+      id: cache-gui
+      with:
+        path: ./chia-blockchain-gui
+        key: ${{ runner.os }}-chia-blockchain-gui-${{ steps.gui-ref.outputs.GUI_REF }}
```

Comment:
I think this will share the same runner.os as the rpm build (linux). Should these be separated into variants with `-rpm` and `-deb` ?

Reply:
I compared diff of GUI build output between fedora/ubuntu and found that checksum of `chia-blockchain-gui/packages/gui/build/renderer/loadable-stats.json` is different.
This is because `loadable-stats.json` contains full path to a js file and this is rather install path dependant than rpm/deb.
But yes, your suggestion makes sense. Adding `-rpm` and `-deb` to cache key indirectly resolves this issue.